plugins/backstage-plugin-coder/src/testHelpers/server.ts

```diff
@@ -8,14 +15,60 @@ import {
   mockWorkspaceBuildParameters,
 } from './mockCoderAppData';
 import {
+  mockBearerToken,
   mockCoderAuthToken,
   mockBackstageProxyEndpoint as root,
 } from './mockBackstageData';
 import type { Workspace, WorkspacesResponse } from '../typesConstants';
 import { CODER_AUTH_HEADER_KEY } from '../api';
 
+type RestResolver<TBody extends DefaultBodyType = any> = ResponseResolver<
```

Comment:
Not a huge fan of needing to bust out the `any` type for these MSW helper types, but it's what MSW itself is using, and trying to make things more specific got nasty